blosc/blosc2.c

```diff
@@ -3208,6 +3209,7 @@ int init_threadpool(blosc2_context *context) {
   int rc2;
 
   /* Initialize mutex and condition variable objects */
+  pthread_spin_init(&context->count_spin, PTHREAD_PROCESS_SHARED);
```

Comment:
`PTHREAD_PROCESS_SHARED` why we need this flag?

Reply:
> `PTHREAD_PROCESS_SHARED` why we need this flag?

int pthread_spin_init(pthread_spinlock_t *lock, int pshared);

The pshared must be one of the following.
PTHREAD_PROCESS_PRIVATE/PTHREAD_PROCESS_SHARED
The 1st one has limited to the threads that are forked from the same process.
The 2nd one could be obtained by all the threads.

Reference Link: https://man7.org/linux/man-pages/man3/pthread_spin_init.3.html